content/setup/proxy_reverse.md

```diff
@@ -45,3 +45,35 @@ location / {
     chunked_transfer_encoding off;
 }
 ```
+
+# Apache2
+
+Example Apache2 reverse proxy configuration:
+
+```
+<VirtualHost *:80>
+
+	ServerName drone.example.com
+	ServerAlias drone
+
+	ServerAdmin webmaster@localhost
+
+	ErrorLog ${APACHE_LOG_DIR}/drone-error.log
+	CustomLog ${APACHE_LOG_DIR}/drone-access.log vhost_combined
+
+	#-#ProxyPass / http://127.0.0.1:8080/
+	#-#ProxyPassReverse / http://127.0.0.1:8080/
+	ProxyPass         / http://localhost:8080/ nocanon
+	ProxyPassReverse  / http://localhost:8080/
+	RequestHeader set X-Forwarded-Proto "http"
+	RequestHeader set X-Forwarded-Port "80"
+	ProxyRequests     Off
+	#ProxyRequests     On	#not needed for reverse proxy
```

Comment:
Pulled this detail from the apache2 man page:
   https://httpd.apache.org/docs/current/mod/mod_proxy.html#forwardreverse
after thinking it was needed.
Happy to elide it.